src/Goerli-DssSpell.sol

```diff
@@ -42,256 +42,83 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
-        DssExecLib.authorize(addr, NEW_MCD_ESM);
+        // Decrease the UNIV2WBTCETH-A Stability Fee from 3% to 2%.
+        DssExecLib.setIlkStabilityFee("UNIV2WBTCETH-A", TWO_PCT_RATE, true);
 
-        // MCD_CLIP_WSTETH_A
-        addr = DssExecLib.getChangelogAddress("MCD_CLIP_WSTETH_A");
-        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
-        DssExecLib.authorize(addr, NEW_MCD_ESM);
+        // Decrease the UNIV2USDCETH-A Stability Fee from 2.5% to 1.5%.
+        DssExecLib.setIlkStabilityFee("UNIV2USDCETH-A", ONE_PT_FIVE_PCT_RATE, true);
 
-        // MCD_CLIP_WBTC_B
-        addr = DssExecLib.getChangelogAddress("MCD_CLIP_WBTC_B");
-        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
-        DssExecLib.authorize(addr, NEW_MCD_ESM);
+        // Decrease the GUNIV3DAIUSDC2-A Stability Fee from 0.5% to 0.25%.
+        DssExecLib.setIlkStabilityFee("GUNIV3DAIUSDC2-A", ZERO_PT_TWO_FIVE_PCT_RATE, true);
 
-        // MCD_CLIP_WBTC_C
-        addr = DssExecLib.getChangelogAddress("MCD_CLIP_WBTC_C");
-        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
-        DssExecLib.authorize(addr, NEW_MCD_ESM);
+        // Decrease the TUSD-A Stability Fee from 1% to 0%.
+        DssExecLib.setIlkStabilityFee("TUSD-A", ZERO_PCT_RATE, true);
 
-        // MCD_CLIP_PSM_GUSD_A
-        addr = DssExecLib.getChangelogAddress("MCD_CLIP_PSM_GUSD_A");
-        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
-        DssExecLib.authorize(addr, NEW_MCD_ESM);
 
-        // MCD_CLIP_GUNIV3DAIUSDC2_A
-        addr = DssExecLib.getChangelogAddress("MCD_CLIP_GUNIV3DAIUSDC2_A");
-        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
-        DssExecLib.authorize(addr, NEW_MCD_ESM);
+        /// DIRECT-AAVEV2-DAI (Aave D3M) Target Borrow Rate Decrease
 
-        // MCD_VAT
-        addr = DssExecLib.getChangelogAddress("MCD_VAT");
-        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
-        DssExecLib.authorize(addr, NEW_MCD_ESM);
+        // Decrease the DIRECT-AAVEV2-DAI Target Borrow Rate from 3.75% to 3.5%.
+        //   NOT AVAILABLE ON GOERLI
 
-        DssExecLib.setChangelogAddress(MCD_ESM, NEW_MCD_ESM);
-        DssExecLib.setChangelogVersion("1.10.0");
+        /// Maximum Debt Ceiling Changes
 
+        // Decrease the GUNIV3DAIUSDC1-A Maximum Debt Ceiling from 500 million DAI to 100 million DAI.
+        DssExecLib.setIlkAutoLineParameters("GUNIV3DAIUSDC1-A", 100 * MILLION, 10 * MILLION, 8 hours);
 
-        // -----------
-        // Deauthorize the existing lerp to prevent additional overwrites of hump.
-        // https://vote.makerdao.com/executive/template-executive-vote-temporarily-prevent-surplus-flap-auctions-and-mkr-burn-january-24-2022?network=mainnet#proposal-detail
-        DssExecLib.deauthorize(DssExecLib.vow(), SB_LERP);
+        // Increase the GUNIV3DAIUSDC2-A Maximum Debt Ceiling from 500 million DAI to 750 million DAI.
+        DssExecLib.setIlkAutoLineParameters("GUNIV3DAIUSDC2-A", 750 * MILLION, 10 * MILLION, 8 hours);
```

Comment:
I think you are missing this:
"Increase the GUNIV3DAIUSDC2-A Target Available Debt (gap) from 10 million DAI to 50 million DAI."

Reply:
96996a792ab814094b2cc4c1b9baf52d95aec0e5